frontend/pages/ManageControlsPage/Scripts/components/ScriptDetailsModal/ScriptDetailsModal.tsx

```diff
@@ -203,13 +204,19 @@ const ScriptDetailsModal = ({
             >
               <Icon name="download" />
             </Button>
-            <Button
-              className={`${baseClass}__action-button`}
-              variant="icon"
-              onClick={onDelete}
-            >
-              <Icon name="trash" color="ui-fleet-black-75" />
-            </Button>
+            <GitOpsModeTooltipWrapper
+              position="bottom"
+              renderChildren={(disableChildren) => (
```

Comment:
Do we want to be consistent and name these all as `disabledChildren` / `dC` but not either?